combat-trainer.lic

```diff
@@ -65,6 +70,27 @@ class SetupProcess
 
   private
 
+  def blow_warhorn()
+    return unless @warhorn
+    return if Time.now - @last_warhorn < 600
+
+    case bput("tap my #{@warhorn}", 'inside your', 'that you are wearing', 'I could not find what you were referring to.')
+    when 'that you are wearing'
+        get_verb = "remove"
+        store_verb = "wear"
+    else
+        get_verb = "get"
+        store_verb = "stow"
+    end
+
+    if ['You remove','You get'].include? bput("#{get_verb} my #{@warhorn}", 'You get', 'You remove', 'What were you referring to', 'You need a free hand', 'Remove what')
```

Comment:
I prefer the `case`/`when` syntax for this kinds of situations. What do you think?

Reply:
Done!